### PR TITLE
Modified signTransaction logic and implemented feePayerSignTransaction

### PR DESCRIPTION
**Pull request by jimni1222**
## Proposed changes

This PR introduces modification signing logic and newly implemented functions.

1. signTransaction is modified for handling RLP encoded raw transaction and appending.

2. feePayerSignTransaction is implemented for fee payer.

3. signTransaction and feePayerSignTransaction function remove duplication of signatures.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/caver-js/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/caver-js)
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

refer #117 

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: packages/caver-klay/caver-klay-accounts/src/index.js [hunk 1]
```diff
@@ -68,6 +68,70 @@ function coverInitialTxValue(tx) {
   return tx
 }
 
+function resolveArgsForSignTransaction(args) {
```
Comment (kjhman21) [inline packages/caver-klay/caver-klay-accounts/src/index.js]: Please add description of this function.

File: packages/caver-klay/caver-klay-accounts/src/index.js [hunk 2]
```diff
@@ -68,6 +68,70 @@ function coverInitialTxValue(tx) {
...
+    callback = args[2]
+  }
+
+  // For handling when callback is undefined.
+  callback = callback || function () {}
+
+  return { tx, privateKey, callback }
+}
+
+function resolveArgsForFeePayerSignTransaction(args) {
```
Comment (kjhman21) [inline packages/caver-klay/caver-klay-accounts/src/index.js]: Please add description of this function.

File: packages/caver-klay/caver-klay-accounts/src/index.js (hunk 1 above)
Comment (jimni1222) [reply packages/caver-klay/caver-klay-accounts/src/index.js]: Description for resolveArgsForSignTransaction is added

File: packages/caver-klay/caver-klay-accounts/src/index.js (hunk 2 above)
Comment (jimni1222) [reply packages/caver-klay/caver-klay-accounts/src/index.js]: Description for resolveArgsForFeePayerSignTransaction is added :)

File: packages/caver-klay/caver-klay-accounts/src/index.js [hunk 3]
```diff
@@ -460,80 +538,105 @@ Accounts.prototype.getLegacyAccount = function getLegacyAccount(key) {
   return { legacyAccount: account, klaytnWalletKeyAddress }
 }
 
+/**
+ * signTransaction signs to transaction with private key.
```
Comment (kjhman21) [inline packages/caver-klay/caver-klay-accounts/src/index.js]: It seems that if we already have signatures in the tx, the new signature will be appended. If it is right, please describe more about that.

Comment (jimni1222) [reply packages/caver-klay/caver-klay-accounts/src/index.js]: The description for appending was added :)

**Pull request merged**